src/audits/FocusableElementNotVisibleAndNotAriaHidden.js

```diff
@@ -42,7 +42,8 @@ axs.AuditRules.addRule({
         }
         // Ignore elements which have a negative tabindex and no text content,
         // as they will be skipped by assistive technology
-        if (axs.properties.findTextAlternatives(element, {}).trim() === '')
+        var alternative = axs.properties.findTextAlternatives(element, {});
```

Comment:
Good catch! I just submitted a separate PR for this same issue, not having seen this, sorry.